src/words/TokTok.jl

```diff
@@ -0,0 +1,178 @@
+# Tok-Tok Tokenizer
+# Replace non-breaking spaces with normal spaces.
+
+const NON_BREAKING = ("\u00A0",) => " "
+
+# Pad some funky punctuation.
+const FUNKY_PUNCT_1 = string.(Tuple("،;؛¿!])}»›”؟¡%٪°±©®।॥…"))
+# Pad more funky punctuation.
+const FUNKY_PUNCT_2 = string.(Tuple("[“‘„‚«‹「『"))          
+# Pad En dash and em dash
+const EN_EM_DASHES = ("–—")
```

Comment:
```suggestion
const EN_EM_DASHES = ("–—",)
```

Should there  be an en-dash here  too?

Reply:
Yeah, there should be. It wasn't in the nltk's version, so I missed it!

Reply:
It is actually missing from the original too.
https://github.com/jonsafari/tok-tok/blob/fd0029cad4f3eed2ae229d835c2c77ea39931fd1/tok-tok.pl#L72

Maybe @jonsafari can comment?